IntegrationTests/Braintree-API-Integration-Specs/BraintreeApplePay_IntegrationTests.swift

```diff
@@ -6,7 +6,7 @@ import PassKit
 class BraintreeApplePay_IntegrationTests: XCTestCase {
 
     func testTokenizeApplePayPayment_whenApplePayEnabledInControlPanel_returnsANonce() {
-        let apiClient = BTAPIClient(authorization: BTIntegrationTestsConstants.sandboxTokenizationKey)!
+        let apiClient = BTAPIClient(authorization: BTIntegrationTestsConstants.sandboxTokenizationKey)
```

Comment:
Since we are constructing the API Client under the hood in the Apple Pay client is there a reason we need to define it here. Couldn't we instead pass `BTIntegrationTestsConstants.sandboxTokenizationKey` into the Apple Pay Client on line 10 and remove `applePayClient.apiClient = apiClient`? If so we should be able to do that in both tests here.

Reply:
ahh yep great catch - we no longer need to define and instantiate `BTAPIClient` here, i've updated this in d45e3df5a6be30c47e75f182aaa27c67aa4769c6